Mathlib/Topology/Algebra/Module/PerfectPairing.lean

```diff
@@ -0,0 +1,70 @@
+/-
+Copyright (c) 2025 Yaël Dillies, Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Andrew Yang
+-/
+import Mathlib.LinearAlgebra.BilinearMap
+import Mathlib.Topology.Algebra.Module.LinearMap
+
+/-!
+# Continuous perfect pairings
+
+This file defines continuous perfect pairings.
+
+For a topological ring `R` and two topological modules `M` and `N`, a continuous perfect pairing is
+a continuous bilinear map `M × N → R` that is bijective in both arguments.
+
+Note that this does **not** imply that the continuous dual of `M` is `N` and vice-versa (one would
+instead need "homeomorphic in both arguments").
```

Comment:
I think we should include Anatole's / your example of `M := WeakDual R N` here with a careful explanation: it will definitely save some future person a few minutes.

Reply:
What do you think of this explanation?

Reply:
I think some more detail would help. I've made a suggestion elsewhere.